docs/apv.md

```diff
@@ -562,6 +562,7 @@ Onderstaande regels tellen alleen op Narcopolis
 1. Het is verplicht om te landen op een helipad, een landingsbaan of in een straal van 50 meter van een helipad/landingsbaan.
 2. De sea sparrow en dodo mag wel op open water rondom het eiland geland worden.
 3. Bij overtreding van het feit beschreven in lid 1 of lid 2 wordt een straf van de 1e categorie uitgedeeld.
+4. Op lid 1 geldt een uitzondering wanneer het gaat om een noodsituatie, dit is ter beoordeling van het desbetreffende stafflid.
```

Comment:
Hoe moet dit beoordeeld worden? welk bewijs wordt er verwacht als je hier van gebruik wil maken? moet dit vooraf of achter af door een stafflid worden bekeken? en mag alleen een stafflid een noodlanding maken?